src/gpodder/model.py

```diff
@@ -269,9 +269,9 @@ def trimmed_title(self):
                     return title
 
         # "#001: Title" -> "001: Title"
-        if (not self.parent._common_prefix
-                and re.match('^#\d+: ', self.title)
-                and len(self.title)-1 > LEFTOVER_MIN):
+        if (not self.parent._common_prefix and
```

Comment:
not found of that one, because it looks like not applies to all lines.
Better
```
        if (
                not self.parent._common_prefix and
                re.match('^#\d+: ', self.title) and
                len(self.title)-1 > LEFTOVER_MIN):
```